cmd/api/src/database/audit.go

```diff
@@ -79,17 +75,8 @@ func (s *BloodhoundDB) GetAuditLogsBetween(before, after time.Time, offset, limi
 	} else if order == "" && filter.SQLString != "" {
 		result = cursor.Where(filter.SQLString, filter.Params).Find(&auditLogs)
 	} else {
-		result = cursor.Find(&auditLogs)
+		result = cursor.Find(&auditLogs).Count(&count)
```

Comment:
This appears to only create valid counts if no order and no filter are given. Is this intended?

Reply:
The second to last screenshot highlights this issue, showing a count of 0 despite a result being included